checker/checker.js

```diff
@@ -25,14 +24,26 @@ gets fired from github pr creation/update webhook.
 const valid_pr_events = ['opened', 'reopened', 'synchronize'];
 
 async function main (params) {
-  if (!params.pull_request || !valid_pr_events.includes(params.action)) {
+  const isMergeQueue = params.merge_group && params.action === 'checks_requested';
+  if (!isMergeQueue && (!params.pull_request || !valid_pr_events.includes(params.action))) {
     return {
       statusCode: 202,
       body: 'Not a pull request being (re)opened or synchronized, ignoring'
     };
   }
   let github;
   const ow = openwhisk();
+
+  if (isMergeQueue) {
+    const res = await set_green_is_bot(ow, {
+      commit_sha: params.merge_group.head_sha,
```

Comment:
We're missing this one:
```suggestion
      commit_sha: params.merge_group.head_sha,
      installation_id: params.installation.id,
```